docker-app/qfieldcloud/core/views/files_views.py

```diff
@@ -135,25 +138,61 @@ def get(self, request, projectid, filename):
 
     # TODO refactor this function by moving the actual upload and Project model updates to library function outside the view
     def post(self, request, projectid, filename, format=None):
+        # Send traceback to buffer for debugging purposes
+        buffer = io.StringIO()
+        print_stack(limit=20, file=buffer)
```

Comment:
I have a hunch that one of the middlewares is the culprit, so this will hopefully help.

Reply:
I am a bit confused what is the idea here with this empty buffer?

Reply:
Ok, thanks. I think I get it now. We can discuss in further detail when it is in mergeable state.